core/validation/failures.py

```diff
@@ -159,9 +213,30 @@ def __str__(self):
             f'Value "{self.value}" is not a valid enum value.'
         )
 
+    def uc_message_components(self) -> tuple[str, str, str, str]:
+        message = "You must select an option from the list provided. Do not populate the cell with your own content."
+        sheet, section = INTERNAL_TABLE_TO_FORM_LOCATION[self.sheet]
+        column = INTERNAL_COLUMN_TO_FORM_COLUMN[self.column]
+
+        # additional logic for outcomes to differentiate between footfall and non-footfall
+        if sheet == "Outcome_Data" and self.row_values[4] == "Year-on-year % change in monthly footfall":
+            section = "Footfall Indicator"
+            column = "Geography of indicator / measurement"
+
+        # additional logic for risk location
+        if sheet == "Risk Register":
+            if self.row_values[1]:
+                # project risk
+                section = f"Project {int(self.row_values[1].split('-')[2])} Risks"
+            else:
+                # programme risk
+                section = "Programme Risks"
+
+        return sheet, section, column, message
```

Comment:
What are we doing with the value that is wrong? Are we removing all mention of it from the validation?

Reply:
Good point. I don't think I saw it on Elizabeth's UC message but it seems illogical to not include it as it will be useful to use CTRL + F with. I'll look to format it into the message.